### PR TITLE
Support for new Visual Studio Releases.

### PR DESCRIPTION
**Pull request by arielsrv**
Hi! See pull request please!

### CODE REVIEW

File: sample/WebApi.OutputCache.V2.Demo/WebApi.OutputCache.V2.Demo.csproj [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>WebApi.OutputCache.V2.Demo</RootNamespace>
     <AssemblyName>WebApi.OutputCache.V2.Demo</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```
Comment (filipw) [inline sample/WebApi.OutputCache.V2.Demo/WebApi.OutputCache.V2.Demo.csproj]: thanks for the PR - but what is the purpose of this change? ASP.NET Web API can be used with .NET 4.5, so why bump this library?

Comment (arielsrv) [inline sample/WebApi.OutputCache.V2.Demo/WebApi.OutputCache.V2.Demo.csproj]: Is true, can be rollbacked. Thanks for watching.
